modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -1328,11 +1324,9 @@ public Object constructValue() throws LoadException {
                     if (value == null) {
                         try {
                             ReflectUtil.checkPackageAccess(type);
-                            value = type.newInstance();
-                        } catch (InstantiationException exception) {
-                            throw constructLoadException(exception);
-                        } catch (IllegalAccessException exception) {
-                            throw constructLoadException(exception);
+                            value = type.getDeclaredConstructor().newInstance();
+                        } catch (Exception e) {
+                            throw constructLoadException(e);
```

Comment:
This isn't quite equivalent, since it will now wrap a `RuntimeException` in a `LoadException`, but I think that's OK in this case.